src/mechanisms/scram/client.rs

```diff
@@ -485,25 +478,18 @@ where
     }
 }
 
-#[derive(Debug, Ord, PartialOrd, Eq, PartialEq, Copy, Clone)]
+#[derive(Debug, Ord, PartialOrd, Eq, PartialEq, Copy, Clone, thiserror::Error)]
```

Comment:
As we `use` thiserror::Error above I don't think this needs to be fully qualified, does it?

Reply:
I have the habit to qualify all externals types to avoid conflict between `thiserror::Error` and `std::error::Error` for example.